### PR TITLE
Adding The Server ClarryMC

### PR DESCRIPTION
**Pull request by syntax-png**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by syntax-png** [comment]
ClarryMC

File: minecraft_servers/clarrymc/background.png
Comment (DoJapHD) [inline minecraft_servers/clarrymc/background.png:1]: Compress file

File: minecraft_servers/clarrymc/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/clarrymc/background@2x.png:1]: Compress file

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is in maintenance mode; Server has to be publicly available
Reopen PR once server is online 4 every1

**Pull request closed without merging**